README.adoc

```diff
@@ -51,19 +51,13 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart - `git` and `curl` must be installed and available in `$PATH`.
 
-.Generate sample Gradle project and run Gatling simulation
-[source, bash]
-----
-curl -sL https://git.io/Jf2Uk | bash -s ~/sample-gradle-gatling
-----
 
-.Navigate to new project folder and run all simulations
+.Generate sample Gradle project and run Gatling simulation, follow on-screen instructions.
 [source, bash]
 ----
-cd ~/sample-gradle-gatling
-./gradlew gatlingRun
+curl -sL https://git.io/JJj3A | bash -s ~/gradle-gatling-sample
```

Comment:
But this is a official GitHub product. Why is there anyy good need to turn brief and consice cmd line into long lumsy string that is incomfortable to paste into shell ?

Reply:
1. Official or not, the shortener hides the real link
2. It can be reassuring that the URL links to our own repository file.
3. Copying some more characters is not an issue (select all the block copy, paste, done) and we are not under pressure for the message length (tweet or SMS)


I can return the question: why using another service, another dependency when we can avoid it without downside?

Reply:
Here are the reasons why this is done that way,

1. Having a long strings to be copy \ pasted is always a sign of documentation smell (badly created docs).
* it's uncomfortable to select and then copy, because needs more mouse movement, depending on the size of the it maybe a pain at all)
* it's uncomfortable to paste, because the resulting cmd line may break an bash history, it hard to nagivate in history when pasted code needs to be changed (missing letter, for example).
Having to copy \ paste long string is not helping peoples to create sample project quick and comfortable, but instead making them a sort of angry when dealing with mouse selection and resulting long lines in bash history.

2. Then, I tend to disagree about the sentence regarding another service or another dependency.
This URL shortener is the same GitHub, just another componet, like PR creating, like Wiki etc. Theres' no redirection to untrusted sites because Git.IO doesn't allow to paste links going to sites different from GitHub. Everything regarding security is controlled by GitHub itself, so if one trusts GitHub - why doens't he trust GitHub URL shortener. ?

3. Users who will be generating are already on GitHub and they already trust official Gatling repository. They will not take much care about exact URL they are pasting. So putting into docs a long clumsy string just to prove it goes to GitHub (meanwhie git.io does it itself) - doesn't make a lot of benefits. Furthemore, a TIP could be added to explain what is git.io and where it exactly goes.

Keeping code blocks in GitHub README is always a good expirience for users and using URL shortener will UX benefits for those who will be generating sample projects using the script from this PR.

Reply:
I am not convinced.

> Having a long strings to be copy \ pasted is always a sign of documentation smell (badly created docs).

`curl | bash` is wayyy smeller than long strings to be copied. In fact, this pattern is present only to hide the very long strings (and steps) to be copied. So, here we have the "documentation smell".

> This URL shortener is the same GitHub, just another componet,

git.io is indeed a different service and can be down while the other parts are not (this point only to show that it is a new dependency). But I was not aware that it does not allow to paste links going to sites different from GitHub. It mitigates that security concern but users may be as (mis)informed as I.

> Furthemore, a TIP could be added to explain what is git.io and where it exactly goes.

If we have to explain it only for a single link, we should not use it at all.

Perhaps can we agree to:
 * add a direct link to the script
   there is a sentence before: "The plugin provides bootstrap script that creates sample project with:", where we can add a link from "bootstrap script" to the actual "boostrap.sh" script ?
 * Add a quick explanation:
    ```
    To directly execute the https://github.com/gatling/gatling-gradle-plugin/blob/master/bootstrap.sh[bootstrap script], run the following command:
    ```


Note: the git.io link will not be valid after the branch deletion as it links to the `boostrap_project` branch.